code/apps/Managed Software Center/Managed Software Center/ja.lproj/Localizable.strings

```diff
@@ -1,8 +1,8 @@
 ﻿/* Multiple Updates message */
-"%s pending updates" = "%s個のアップデートを保留しています";
+"%@ pending updates" = "アップデートを%@個保留しています";
 
```

Comment:
munki-notifier, on the other hand, _is_ a "normal" Cocoa app and does use %@ for the string object placeholders.